src/rdo.rs

```diff
@@ -823,6 +823,23 @@ pub fn rdo_tx_type_decision(
   best_type
 }
 
+pub fn get_sub_partitions<'a>(four_partitions: &[&'a BlockOffset; 4],
```

Comment:
@lu-zero, Can you check my rust code to factor out the block of code seem correct?

Reply:
`get_sub_partitions` ? It is fine

Reply:
Ok- thanks!